database/schema.sql

```diff
@@ -842,3 +842,47 @@ ALTER TABLE dbo.sccv_person_case_status_answers
   
 ALTER TABLE dbo.sccv_person_case_status_answers
 	ADD COLUMN end_date timestamp;
+
+  --MASH tables
+  CREATE TABLE IF NOT EXISTS dbo.REF_MASH_REFERRALS
+(
+  id bigint GENERATED BY DEFAULT AS IDENTITY NOT NULL PRIMARY KEY,
+  referrer varchar(256),
+  requested_support varchar(256),	
+  worker_id bigint,
+  created_at timestamp without time zone,
+  referral_doc_url varchar(256),
+  stage varchar(30),
+  referral_category varchar(256),
+  initial_decision varchar(256),
+  initial_decision_referral_category varchar(256),
+  initial_decision_urgent_contact bool,
+  screening_decision varchar(256),
+  screening_decision_urgent_contact bool,
+  final_decision varchar(256),
+  final_decision_referral_category varchar(256),
+  final_decision_urgent_contact bool,

```

Comment:
initial, screening and final all need a created_at, so initial_decision_created_at which in the code will be a DateTime, so a timestamp would make sense I expect as the type

Reply:
will add the new fields in upcoming commit